deploy/ui/form.json

```diff
@@ -1619,6 +1619,133 @@
               ]
             }
           ]
+        },
+        {
+          "label": "Process Instance Migration",
+          "fields": [
+            {
+              "label": "Image",
+              "required": false,
+              "jsonPath": "$.spec.objects.processMigration.image",
+              "description": "The image to use for Process Instance Migration.",
+              "type": "text"
+            },
+            {
+              "label": "Image tag",
+              "required": false,
+              "jsonPath": "$.spec.objects.processMigration.imageTag",
+              "description": "The image tag to use for Process Instance Migration.",
+              "type": "text"
+            },
+            {
+              "label": "Database",
+              "type": "fieldGroup",
+              "visible": true,
+              "jsonPath": "$.spec.objects.processMigration.database",
+              "fields": [
+                {
+                  "label": "Database type",
+                  "type": "dropDown",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.database.type",
+                  "originalJsonPath": "$.spec.objects.processMigration.database.type",
+                  "fields": [
```

Comment:
I think the "options" are missing. It should be possible to choose between:
* h2
* mysql
* postgresql
* external

Reply:
It will be referred from CRD changes (https://github.com/xiezhang7/kie-cloud-operator/blob/RHPAM-2435/deploy/crds/kieapp.crd.yaml#L704).

Reply:
Thanks